docs/source/using/profiles.adoc

```diff
@@ -8,29 +8,50 @@ include::variables.adoc[]
 
 toc::[]
 
-[[pofile-overview]]
+[[profile-overview]]
 == Overview
 
-{project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+The profile feature allows creating multiple instances of {project}. 
 
-[[creating-profile]]
+A profile is an instance of the {project} VM along with all of its configuration and state.
+
+Each {project} profile can be created with its own configuration (memory, CPU, disk size, add-ons, and so on) and completely independent of other profiles.
+If you have common configuration options you want to apply to all profiles, you have to do this manually per profile.
+See xref:../using/profiles.adoc#setting-global-config-profiles[Setting Global Configuration Options].
+
+The current profile being used is called the active profile.
+By default {project} commands will run against the active profile.
+You can run the xref:../command-ref/minishift_profile_list.adoc#[`minishift profile list`] command to determine the active profile.
+
+The profile feature introduced a global `--profile` flag and xref:../command-ref/minishift_profile.adoc#[commands] for listing, deleting and setting the active profile.
+
+The `--profile` global flag can be specified with any {project} command to run it against the specific profile, bypassing the currently active profile.
+
+NOTE: Commands are run against the default profile if you are not using the `--profile` flag and have not set a different active profile.
+
+[[creating-profiles]]
 == Creating Profiles
 
-You can create a new instance of {project} with xref:../command-ref/minishift_start.adoc#[`minishift start --profile`] command.
+You can create a new instance of {project} with the xref:../command-ref/minishift_start.adoc#[`minishift --profile PROFILE_NAME start`] command.
```

Comment:
Again, there is a disconnect here. We are talking about profiles and how to create them, but then the section starts with "you can create a new instance of {project}". I think we need to motiviate this better.

"There are two ways to create a new profile, explicitly or implicitly. 

=== Explicit Profile Creation 

bla bla bla

    $ minishift profile set my-profile

=== Implicit Profile creation

bla bla bla

    $ minishift --profile my-profile start

Reply:
@hferentschik `minishift profile set my-profile` it does not create any profile but sets it as active profile. I think there a difference between creating a profile(where artifacts gets created) and setting active profile.

Reply:
I think you have mistakenly reversed the use of `Explicit` and `Implicit` above.

Reply:
I guess we need to separate the section between creating a profile (implicitly) and starting of a profile. Let see what I came back with.